src/components/textarea/textarea.component.tsx

```diff
@@ -116,227 +117,266 @@ export interface TextareaProps
   warning?: boolean | string;
 }
 
-export const Textarea = ({
-  autoFocus,
-  fieldHelp,
-  label,
-  size,
-  children,
-  characterLimit,
-  enforceCharacterLimit = true,
-  warnOverLimit = false,
-  onChange,
-  disabled = false,
-  labelInline,
-  labelAlign,
-  labelHelp,
-  labelSpacing,
-  inputIcon,
-  id: idProp,
-  error,
-  warning,
-  info,
-  name,
-  readOnly = false,
-  placeholder,
-  expandable = false,
-  rows,
-  cols,
-  validationOnLabel = false,
-  adaptiveLabelBreakpoint,
-  inputWidth,
-  maxWidth,
-  labelWidth = 30,
-  tooltipPosition,
-  value,
-  "data-component": dataComponent,
-  "data-element": dataElement,
-  "data-role": dataRole,
-  helpAriaLabel,
-  ...props
-}: TextareaProps) => {
-  const { validationRedesignOptIn } = useContext(NewValidationContext);
-  const computeLabelPropValues = <T,>(prop: T): undefined | T =>
-    validationRedesignOptIn ? undefined : prop;
+let deprecateInputRefWarnTriggered = false;
 
-  const { current: id } = useRef(idProp || guid());
+export const Textarea = React.forwardRef(
+  (
+    {
+      autoFocus,
+      fieldHelp,
+      label,
+      size,
+      children,
+      characterLimit,
+      enforceCharacterLimit = true,
+      warnOverLimit = false,
+      onChange,
+      disabled = false,
+      labelInline,
+      labelAlign,
+      labelHelp,
+      labelSpacing,
+      inputIcon,
+      id: idProp,
+      error,
+      warning,
+      info,
+      name,
+      readOnly = false,
+      placeholder,
+      expandable = false,
+      rows,
+      cols,
+      validationOnLabel = false,
+      adaptiveLabelBreakpoint,
+      inputWidth,
+      maxWidth,
+      labelWidth = 30,
+      tooltipPosition,
+      value,
+      "data-component": dataComponent,
+      "data-element": dataElement,
+      "data-role": dataRole,
+      helpAriaLabel,
+      inputRef,
+      ...props
+    }: TextareaProps,
+    ref
```

Comment:
```suggestion
    ref: React.ForwardedRef<HTMLInputElement>
```

Reply:
strangely, TS doesn't seem to need this, even though when I hover over it it says React.ForwardedRef<unknown> (but the way it's used implies it's an `HTMLInputElement` inside, as it's passed as `ref` to `Input` which has that typed correctly). Not sure what's going on but I agree it's better to be explicit if only so it's easier for us to see in future what the intended type is.